float64[16] pose # 4*3 mtx
uint32   id
bool   isFixed
